prime-router/docs/universal-pipeline/convert.md

```diff
@@ -7,26 +7,40 @@ The convert step is executed when the [Azure AQS message](README.md#aqs-usage) r
+The first and most significant aspect of the convert step is the conversion of a message into the FHIR format. The 
+process of this conversion depends on the incoming message's format:
+
+- If the incoming message is HL7v2, ReportStream uses the [LinuxForHealth HL7 to FHIR Converter library](https://github.com/LinuxForHealth/hl7v2-fhir-converter#linuxforhealth-hl7-to-fhir-converter) with custom configurations from the 
+- [fhir_mapping folder](https://github.com/CDCgov/prime-reportstream/tree/master/prime-router/metadata/fhir_mapping/hl7) 
+-to handle the conversion. See the 
+- [HL7v2-to-FHIR configurations](../design/design/transformations.md#HL7v2-to-FHIR-configurations) section for more 
+- details.
 - If the incoming message is FHIR, then there's nothing to be done
 - Apply Sender transforms specified by Sender configuration 
 
-> See [HL7v2-to-FHIR Transformations](../design/design/transformations.md#hl7v2-to-fhir-transformations) for more information regarding configuration of HL7 to FHIR transformations.
+> See [HL7v2-to-FHIR Transformations](../design/design/transformations.md#hl7v2-to-fhir-transformations) for more 
+> information regarding configuration of HL7 to FHIR transformations.
 
-> Currently, no other input formats besides HL7v2 and FHIR are supported in the Universal Pipeline. In order to support new formats, they would need to be transformed to FHIR by some new method.
+> Currently, no other input formats besides HL7v2 and FHIR are supported in the Universal Pipeline. In order to support 
+> new formats, they would need to be transformed to FHIR by some new method.
 
 ## Sender Transforms
-After the conversion of an Item to the FHIR format, an additional transform can run on the resulting FHIR bundle. This transform is called a "Sender Transform" because it is used to make sender-specific adjustments to the FHIR bundle.
+After the conversion of an Item to the FHIR format, an additional transform can run on the resulting FHIR bundle. This 
+transform is called a "Sender Transform" because it is used to make sender-specific adjustments to the FHIR bundle.
 
-Sender Transforms are completely optional, but ReportStream has created a default sender transform that can be extended or used directly (`prime-router/metadata/fhir_transforms/senders/default-sender-transform.yml`). This is a good place to start to understand what kind of transforms a Sender may want performed on their messages.
+Sender Transforms are completely optional, but ReportStream has created a sender transform that mimics the behavior of 
+original pipeline (`prime-router/metadata/fhir_transforms/senders/default-sender-transform.yml`). This is a good place 
```

Comment:
Should this be refactored to be consistent with the yml files?

Reply:
> I'm not sure what you mean
should the referred schema file name be "original-pipeline-transforms.yml"?
"original pipeline (`prime-router/metadata/fhir_transforms/senders/default-sender-transform.yml`). This is a good place "